### ISSUE TITLE
[Fix] [Visualization] Trend plot and estimate_accuracy plot have xrange() first term set to 0

### ISSUE THREAD

**Issue by andrybicio** [lisphilar/covid19-sir#570]
## Summary
When we plot the trend plot, the recovered individual number (on x-axis) starts from 0, hence the curve is not centered. See image1.

 When we plot the estimate_accuracy(phase='phase') for any phase, we observe the same problem for the latest two comparison curves (see image2).

## Codes 
```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
population_data = data_loader.population()
# Scenario analysis
National_scenario = cs.Scenario(jhu_data, population_data, country="Italy", province= None, tau=1440)
# Fix the first date of records
National_scenario.first_date = "04May2020"
# Fix the last date of records
National_scenario.last_date = "20Oct2020"
National_scenario.trend(set_phases=True)
National_scenario.summary()
```

## Outputs

![download](https://user-images.githubusercontent.com/56033694/105861528-19d2cd00-5fef-11eb-94c8-80571f0c58a0.png)

![download (2)](https://user-images.githubusercontent.com/56033694/105860807-4508ec80-5fee-11eb-8acd-00653a678200.png)

## Environment
- CovsirPhy version: 2.15.0
- Python version: 3.7.9
- Installation: pip
- System: Ubuntu

**Comment by lisphilar** [comment]
With #574, the figure of trend analysis will be
![Figure_1](https://user-images.githubusercontent.com/7270139/106353056-2591fc80-632b-11eb-8eec-efb6822ed90b.png)

We can change the lower limit of x-axis manually with `Scenario.trend(xlim=(1000, None))`.

**Comment by lisphilar** [comment]
This is related to #568 and #575.

**Comment by lisphilar** [comment]
With #576, the lower x-axis limit of `Scenario.estimate_accuracy()` figure was updated.
![Figure_1](https://user-images.githubusercontent.com/7270139/106353570-e2d22380-632e-11eb-8ec4-225b647be949.png)

**Comment by andrybicio** [comment]
I think it has been fixed. Then I'm closing it. Thank you
